processing/src/main/java/io/druid/query/QueryContexts.java

```diff
@@ -111,7 +111,7 @@
   public static <T> long getTimeout(Query<T> query, long defaultTimeout)
   {
     final long timeout = parseLong(query, TIMEOUT_KEY, defaultTimeout);
-    Preconditions.checkState(timeout >= 0, "Timeout must be a non negative value, but was [%d]", timeout);
+    Preconditions.checkState(timeout >= 0, "Timeout must be a non negative value, but was [%s]", timeout);
```

Comment:
Surprisingly it's not supported by Preconditions. Check https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Preconditions.html